src/Main.hs

```diff
@@ -190,15 +197,15 @@ findUserNick users uid =
 
 lookupFlows :: FlowMap -> ParamName Organisation -> [ParamName Flow]
 lookupFlows flowsMap org =
-  maybe [] id $ HM.lookup org flowsMap
+  maybe [] (L.sort . NE.toList) $ HM.lookup org flowsMap
```

Comment:
If you don't sort, the order they come from the API is the what the user has dragged them to. I think that could be a sensible order to keep, for example if passing the output to `head` or further filtering/processing with `grep`/`awk`/etc